package.json

```diff
@@ -45,8 +45,18 @@
     "type": "git",
     "url": "git://github.com/ramda/ramda.git"
   },
-  "module": "es/index.js",
-  "main": "src/index",
+  "main": "./src/index.js",
+  "exports": {
+    ".": {
+      "require": "./src/index.js",
+      "import": "./es/index.js",
+      "default": "./src/index.js"
+    },
+    "./es/": "./es/",
```

Comment:
Those are here to allow for "deep" imports like `import add from "ramda/es/add.js"`. This isn't needed per se and you could explore hiding those files entirely through listing all allowed entries in this `"exports"` field (which could result in things being accessible like this: `"ramda/add"`), but I've added this for backward compatibility for now.